frontend/src/app/components/register-page/register-page.component.ts

```diff
@@ -10,33 +14,36 @@ import { ValidationConstants } from 'src/entity-models/const-resources/validatio
 })
 export class RegisterPageComponent implements OnInit {
 
-  public name = '';
-  public email = '';
-  public password = '';
   public passwordRepeat = '';
   public hidePass = true;
   public hidePassRepeat = true;
+
+  public userRegister: UserRegisterModel = {};
   public registerForm: FormGroup =  new FormGroup({});
   public nameControl: FormControl;
   public emailControl: FormControl;
   public passwordControl: FormControl;
   public passwordRepeatControl: FormControl;
+  public unsubscribe$ = new Subject<void>();
```

Comment:
Why do you need this?

Reply:
In this file user sending register form to backend=>
this.service.registerUser().pipe(takeUntil(this.unsub..$)).subscribe(...)

Reply:
Yes, but cannot you just subscribe?

Reply:
> Yes, but cannot you just subscribe?
That is a question.
Done

Reply:
> Yes, but cannot you just subscribe?

That is a question.
Done